test/kube2e/helper/testserver.go

```diff
@@ -10,74 +10,13 @@ import (
 )
 
 const (
-	defaultTestServerImage = "quay.io/solo-io/testrunner:v1.7.0-beta17"
+	defaultTestServerImage = "gcr.io/solo-test-236622/testrunner:v1.7.0-beta18"
```

Comment:
Question: is this fine for this image to be in solo-test instead of solo-io, or is it used outside of gloo & solo-projects repo ?
cc: @nfuden @sheidkamp

Reply:
won't this PR eventually make it to k8sgateway ? i.e OSS -> k8sgateway

Reply:
yeah but we have other issues to resolve with that